test/functional/helpers.lua

```diff
@@ -38,7 +38,7 @@ module.nvim_prog = (
 module.nvim_set = (
   'set shortmess+=IS background=light noswapfile noautoindent startofline'
   ..' laststatus=1 undodir=. directory=. viewdir=. backupdir=.'
-  ..' belloff= wildoptions-=pum noshowcmd noruler nomore redrawdebug=invalid')
+  ..' belloff= wildoptions-=pum nohidden joinspaces noshowcmd noruler nomore redrawdebug=invalid')
```

Comment:
did `hidden` actually affect any (lua based) functional test? I only saw a failure for joinspaces before. I wouldn't add it here unless it affects an existing test.